src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/BasicFields.drl

```diff
@@ -44,6 +44,7 @@ rule "Copy basic fields and agenda controller"
         workloadInventoryReport.setCreationDate(vmWorkloadInventoryModel.getScanRunDate());
 
         insert(workloadInventoryReport);
+        kcontext.getKieRuntime().getAgenda().getAgendaGroup("OSFamily").setFocus();
```

Comment:
@m-brophy I think you made a good point suggesting to avoid firing the "Other" OSFamily every time. My last commit removed the DefaultOSFamily rule from "ReasonableDefaults" and moved it to "OSFamily.drl" https://github.com/carlosthe19916/xavier-analytics/blob/0ec1b3d1644cd37d740df3181068ab695f792e71/src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/OSFamily.drl#L15. Wdyt about this change?